docs/parameters.yaml

```diff
@@ -1548,6 +1548,20 @@ deprecated: true
 replacedby: ["Cache.RunLocation", "Origin.RunLocation"]
 components: ["origin", "cache"]
 ---
+name: Xrootd.ConfigFile
```

Comment:
@turetske, I wanted to get your feedback here. Should we immediately split this out into something like `Origin.XrootdConfigFile` and `Cache.XrootdConfigFile` so that we can do custom stuff for fed in a box, or do you not anticipate this being an issue?

Reply:
@turetske a friendly ping on this

And now that I think about it more, will we _ever_ set up a sentinel file for a cache? Maybe this should only be for origins?

Reply:
I think the ticket was to add sentinel file to the origin, but it won't hurt to do the same thing for the cache. I will open a ticket for this.

Reply:
> @turetske, I wanted to get your feedback here. Should we immediately split this out into something like `Origin.XrootdConfigFile` and `Cache.XrootdConfigFile` so that we can do custom stuff for fed in a box, or do you not anticipate this being an issue?

I would argue that this is a config we intended to hide from general admins, and there's not much need to put it at origin/cache level.